pkg/bootkube/bootkube.go

```diff
@@ -110,6 +110,9 @@ func (b *bootkube) Run() error {
 			errch <- err
 		}
 	}()
+        if b.selfHostedEtcd {
```

Comment:
can you please fix the indentation here? `if` needs to be aligned with previous statements.

you can run gofmt to do it.